docs/Operation-Architecture/Orbit_UI/File_Operations/File_Operations.rst

```diff
@@ -28,6 +28,14 @@ To access these operations: 
 
 For each file, you can Download_, `View Info`_, or Delete_.
 
+.. toctree::
```

Comment:
this file is way too short. I think all file operations can be merged into one file

Reply:
and we don't need the toctree

Reply:
My source is plenty long. It's the process that's simple. There are other simple processes. It breaks the document logic to start mashing them together on aesthetic grounds. I am pushing back on this. 

We need the toctree or it won't compile. If it distracts (I'll look at the build) I will hide it.